codersdk/workspaceapps.go

```diff
@@ -83,6 +83,7 @@ type WorkspaceApp struct {
 	// Healthcheck specifies the configuration for checking app health.
 	Healthcheck Healthcheck        `json:"healthcheck,omitempty"`
 	Health      WorkspaceAppHealth `json:"health"`
+	Group       string             `json:"group"`
```

Comment:
Would it make sense to `omitempty` this, similar to `Icon` and `Command`? My understanding is that the UI is going to collapse multiple `WorkspaceApps` with the same `Group` into one button.